pyscriptjs/tests/integration/test_plugins.py

```diff
@@ -63,12 +63,10 @@ class ExecTestLogger(Plugin):
     async def beforePyScriptExec(self, interpreter, src, pyScriptTag):
         console.log(f'beforePyScriptExec called')
         console.log(f'before_src:{src}')
-        console.log(f'before_id:{pyScriptTag.id}')
```

Comment:
`pyScriptTag` is a `pyodide.ffi.JsProxy` and I was unable to get the `id`.

I have removed this for now...since it wasn't testing something of any significant value I guess...
But of course, if there's a way to resolve this, we should re-add this stuff...

Reply:
pyScriptTag should be unwrapped as an HtmlElement within Python, and not appear like a JsProxy in this context... I was having similar issues with the event listener work, with some methods/attributes on JsProxies not proxying properly. I think we should try to understand why this regression happened.

Reply:
I phrased my response poorly. This is my concern:

> `pyScriptTag` is a `pyodide.ffi.JsProxy` and I was unable to get the `id`. I have removed this for now...

Accessing the `id` attribute here previous worked, and now doesn't. Which means attribute access on JsProxies is broken, in some way we don't understand. Does it need an `await`? Is it specific to async contexts? We don't know, but this previously worked and now doesn't. 

Specific to this code, does this mean users can't access attributes of the `pyScriptTag` at all, and if not, what's the value in passing it?